src/natives/find-window/win/Program.cs

```diff
@@ -76,16 +52,25 @@ private struct IEWindow {
                 return "";
         }
 
-        private static bool EnumWindowsCallback(IntPtr hwnd, IntPtr lParam) {
-            IEWindow ieWindow = FindInIEWindows(hwnd);
-
-            if(ieWindow.url != null) {
-                Console.Out.Write("{0} {1}", hwnd, "ie");
-                Environment.Exit(0);
+        private static string GetClassName(IntPtr hwnd) {
+            int classNameLength;
 
-                return false;
+            StringBuilder className = new StringBuilder(CLASS_NAME_BUFFER_SIZE);
+            
+            classNameLength = GetClassName(hwnd, className, className.Capacity);
+            
+            if(classNameLength > 0) {
```

Comment:
@AndreyBelym, @AlexanderMoskovkin, what do you think, is there a place for 
`return classNameLength > 0 ? className.ToString() : "";`